streaming/src/main/scala/org/apache/spark/streaming/util/WriteAheadLogUtils.scala

```diff
@@ -39,6 +39,7 @@ private[streaming] object WriteAheadLogUtils extends Logging {
 
   val DEFAULT_ROLLING_INTERVAL_SECS = 60
   val DEFAULT_MAX_FAILURES = 3
+  val WAL_CLOSE_AFTER_WRITE = "spark.streaming.writeAheadLog.closeAfterWrite"
```

Comment:
@tdas Noticed that this is going to be super hard. How are we going to differentiate between a receiver WAL and a driver WAL in the write method of FileBasedWAL?